repository/gorm/asset.go

```diff
@@ -0,0 +1,86 @@
+package gorm
```

Comment:
get rid of gorm sub package and just move this under repository

Reply:
Make sense, now everything related lives in package `repository`